### PR TITLE
Adds 80-bundle atlas for RB

### PR DESCRIPTION
**Pull request by arokem**
Addresses #421, but we need to think about how this integrates with work already done in #420.

### CODE REVIEW

File: AFQ/segmentation.py [hunk 1]
```diff
@@ -642,6 +648,33 @@ def segment_afq(self, tg=None):
...
+        if slr_algo == "linear":
+            self.logger.info("Registering tractogram with linear SLR")
+            atlas = self.bundle_dict['whole_brain']
+            self.moved_sl, _, _, _ = whole_brain_slr(
+                atlas, self.tg.streamlines, x0='affine', verbose=False,
+                progressive=self.progressive,
+                greater_than=self.greater_than,
+                rm_small_clusters=self.rm_small_clusters,
+                rng=self.rng)
+        elif slr_algo == "syn":
```
Comment (36000) [inline AFQ/segmentation.py:672]: following the convention in the api.AFQ object, the variable name should be reg_algo and the values should be "slr" or "syn". Also, there should be a way for the api to pass in its own mapping which it already created.

Comment (36000) [reply AFQ/segmentation.py:675]: We might want to just remove SLR from the segmentation class and ask for a mapping. Just like syn, if the user wants to use slr, they should use: https://github.com/yeatmanlab/pyAFQ/blob/9dda145ee06314a45862c07a0137c523276dd5f2/AFQ/registration.py#L476
This reduces redundant code and makes it so all mapping have the same interface.

**Comment by arokem** [comment]
Thanks for the review @36000! I am done with a first pass here, but still need to address your comments.

**Comment by arokem** [comment]
Looks like the change in defaults from 'slr' to 'syn' is causing a regression in one of the tests: https://github.com/yeatmanlab/pyAFQ/pull/431/checks?check_run_id=1146298668#step:6:56, so I am going to investigate a bit more.

**Comment by arokem** [comment]
For now, I suggest that we back off making 'syn' the default registration, until we've had an opportunity to experiment with this more. WDYT @36000 ? Other than that, I think it's ready to merge, once the tests pass.

**Comment by 36000** [comment]
LGTM

**Pull request merged**